lib/runner/extensions/event.command.js

```diff
@@ -75,6 +77,27 @@ module.exports = {
                 run.triggers.assertion.apply(run.triggers, arguments);
             });
 
+            context.on('execution.request', function (cursor, id, requestId, request) {
+                Requester.create({
+                    type: 'http',
+                    source: 'script',  // @todo - get this from the sandbox
+                    cursor: cursor
+                }, run.options.requester, function (err, requester) {
+                    if (err) {
+                        // @todo is this correct? :-o
+                        return run.triggers.exception.apply(run.triggers, arguments);
+                    }
+
+                    var sendId = id + '.' + requestId;
+
+                    requester.on(sendId, run.triggers.io.bind(run.triggers));
+
+                    return requester.request(sendId, new sdk.Request(request), function (err, res) {
+                        context.dispatch('execution.response.' + id, requestId, err, res && (new sdk.Response(res)));
```

Comment:
@shamasis If there's an error and no response is available, do you want to send an empty response object inside the sandbox?